responders/src/UI/embc-responder/src/app/core/services/customValidation.service.ts

```diff
@@ -79,40 +79,40 @@ export class CustomValidationService {
     /**
```

Comment:
is there anything left here?

Reply:
yes,  I am reusing from the registrant so removed some registrant specific validations